test/perf/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+CONDUIT_IMAGE=$1
+
+for w in workloads/*.sh; do
+
+  ./run-workload.sh $CONDUIT_IMAGE "$w"
```

Comment:
The logs printed to the console could be separated a bit better between workloads, for example add a new line between them or make the workload name stand out a bit more by prefixing `-- `. Also the message is very repetitive, I think printing it once at the start would be enough:
```
When interpreting test results, please take into account, that if built-in plugins are used, their resource usage is part of Conduit's usage too.
```

Reply:
Agreed. :+1: